sampleconfig/core.yaml

```diff
@@ -585,12 +585,12 @@ chaincode:
     # builders in the order specified below.
     # If you don't need to fallback to the default Docker builder, also unconfigure vm.endpoint above.
     # To override this property via env variable use CORE_CHAINCODE_EXTERNALBUILDERS: [{name: x, path: dir1}, {name: y, path: dir2}]
-    externalBuilders: []
-        # - path: /path/to/directory
-        #   name: descriptive-builder-name
-        #   propagateEnvironment:
-        #      - ENVVAR_NAME_TO_PROPAGATE_FROM_PEER
-        #      - GOPROXY
+    externalBuilders:
```

Comment:
Above, it would be worthwhile to comment about the new out-of-the-box ccaas, in addition to the default Docker builder.